orderbook/src/api.rs

```diff
@@ -19,3 +23,24 @@ pub fn handle_all_routes(
             .or(order_by_uid),
     )
 }
+
+#[derive(Serialize)]
+#[serde(rename_all = "camelCase")]
+struct Error<'a> {
```

Comment:
This has been moved to here from handler.rs  because both filter.rs and handler.rs need it.